pgfinder/__init__.py

```diff
@@ -1,4 +1,17 @@
 """Package initialisation"""
-from pgfinder.logs.logs import setup_logger
+import logging
+from pgfinder.pgio import read_yaml
+from pgfinder.logs.logs import setup_logger, LOGGER_NAME
+from pgfinder.utils import dict_to_decimal
 
 LOGGER = setup_logger()
+LOGGER = logging.getLogger(LOGGER_NAME)
+
+PARAMETERS_FILE = "config/parameters.yaml"
```

Comment:
I like how these variable names in in caps. Is that a PEP thing?

Reply:
Yes, its the convention for naming CONSTANTS, described under [PEP008 - Function and variable names](https://peps.python.org/pep-0008/#function-and-variable-names).

[Pylint C0101](http://pylint-messages.wikidot.com/messages:c0103) is reported if you get it wrong (although I haven't actually started applying Pylint to any of this code its just a habit/convention I've managed to internalise).